hrs-portlets-webapp/src/main/webapp/WEB-INF/tags/hrs/pagerNavBar.tag

```diff
@@ -9,15 +9,15 @@
 <div class="flc-pager-${position} dl-pager-navbar" style="display:none">
   <ul class="dl-pager-bar fl-pager-ui uw-pager-bar">
     <li class="flc-pager-previous dl-pager-previous">
-      <a href="javascript:;" title="Previous Page" class="btn btn-default">Previous</a>
+      <a href="#" title="Previous Page" class="btn btn-default">Previous</a>
```

Comment:
The reason it is `javascript:;` is because it runs some javascript code when you click it. this is by design. Where is it suggested that we change these to `#`? This may break things for fluid, was testing for non-screen readers done on this?